azure-iot-device/azure/iot/device/provisioning/pipeline/pipeline_stages_provisioning.py

```diff
@@ -9,30 +9,21 @@
 from . import pipeline_ops_provisioning
 
 
-class UseSymmetricKeyOrX509SecurityClient(PipelineStage):
+class UseSymmetricKeyOrX509SecurityClientStage(PipelineStage):
```

Comment:
>UseSymmetricKeyOrX509SecurityClientStage [](start = 6, length = 40)

This doesn't seem an extendable classname. Are we just going to keep adding "Or<NewAuthType>Stage" to the end of this class every time there's a new type of auth that it handles?

This could probably wait though - I think this is more indicative of architectural questions regarding Security Clients than anything, and any fixes to this should probably be tied to that.

Reply:
It can become just `UseSecurityClientStage` as the first part (the `SetSecurityClientArgs`) will remain same or should remain same. The second  part will change but unless another security mechanism is done I am not completely sure of the operations inside this stage. So this can wait.

---
In reply to: [295579550](https://github.com/Azure/azure-iot-sdk-python-preview/pull/111#discussion_r295579550) [](ancestors = 295579550)